weiluo/Community_Contribution.Rmd

```diff
@@ -0,0 +1,125 @@
+
```

Comment:
please rename this .Rmd file so that it has no capital letters, ie: "community_contribution.Rmd"